Automattic/jetpack
# Minileven: add options back  as they still exist on sites

#### Changes proposed in this Pull Request:

This is a follow-up to #14714.
After removing those options, we started seeing errors like this one on sites:
PHP Warning: Invalid Jetpack option name: wp_mobile_excerpt

Since these Jetpack options are still valid options even if the feature is gone, let's keep them around.

#### Testing instructions:

* Start from a site running Jetpack stable.
* with the Beta plugin, switch from stable to this branch.
* You should see no notices.

#### Proposed changelog entry for your changes:

* N/A
